www/main/settings/components/Setting.jsx

```diff
@@ -31,15 +31,18 @@ const Setting = ({ settingObj, stateVar, stateFunction }) => {
   switch (type) {
     case 'range':
       settingDOM = (
-        <input
-          type="range"
-          data-value={userSettings[stateVar]}
-          value={userSettings[stateVar]}
-          min={min}
-          max={max}
-          step={step}
-          onChange={handleInputChange}
-        ></input>
+        <>
+          <p className='range-value'>{userSettings[stateVar]}</p>
```

Comment:
@Gauravjeetsingh veerji I'm seeing this what's a new window. This is supposed to be removed.

<img width="1455" alt="Screenshot 2023-01-02 at 11 08 18 PM" src="https://user-images.githubusercontent.com/38748298/210263919-f190a845-ceb2-4bbe-96d3-8af246618e55.png">

Reply:
https://github.com/KhalisFoundation/sttm-desktop/issues/1701 opened in separate issue. I think we are good to go with this one